### ISSUE TITLE
Facet field grouping, order and labels

### ISSUE THREAD

**Issue by nielsklazenga** [hiscom/avh-testing#1]
Independently of the AVH test site, I have had a good look at the facet field grouping, ordering and labels, building on the work that Alison did early 2015 (or late 2014?). Not all of the things we suggested then have been implemented, there have been changes in the facets in AVH and, because of the HISPID review we might have different ideas now about the best grouping for different facets. The results are in this Google Doc: https://docs.google.com/document/d/1v8q2LFHozlLFvxWPr6LxjFi2OUkfoTIGKXk9X-3EkgY.

**Comment by acvaughan** [comment]
Thanks Niels. 
I've added comments to the Google doc too. They're not intended to sound defensive; perhaps just some questioning of the need to revisit things that we've spent a long time thinking about already.

**Comment by nielsklazenga** [comment]
Don't worry: they don't sound defensive to me. Just additional useful information that you remember better than I do, as you spent much more time on it then.

**Comment by acvaughan** [comment]
'Remember' is the wrong word. I'm looking here: https://docs.google.com/spreadsheets/d/1q-VgXfsbUzfP4qe1VmC17-gkxpP9T9EAnQvwCdqiwec/edit?usp=sharing

**Comment by nielsklazenga** [comment]
I have been looking for that document, but couldn't remember the URL.

**Comment by acvaughan** [comment]
I haven't done a full comparison of the spreadsheet and your google doc, but it's a start.

**Comment by ryonen** [comment]
I was just looking at the facets spreadsheet and saw your comment re basisOfRecord.

31

Record details

Record type

Record

basisOfRecord

N

We don't need this in AVH because, in theory at least, all our records are preserved specimens. However, there are over 17,000 human observations in AVH at the moment, which we are investigating.

Of the specimens attributed to human observation, over 11,000 of these are at PERTH (with Eileen Croxford having >6,000 to her name, and Gary McCutcheon >4,000) so I thought I’d investigate those. At PERTH ’human observation’ specimens mainly comprise:

1)      photocopies of specimens housed at a regional herbarium (a legacy issue…) [the most common]

2)      sheets made up to house a cibachrome image of a type or other historical specimen in an affixed envelope [far fewer specimens and these unlikely to be retained now that the GPI scans are available].

I just talked to Karina Knight who says that sheets comprising photocopies are recorded as being “photocopies” when PERTH data goes to AVH, and AVH has the following for those sheets:
Basis of record

Human observation
Supplied basis "DrawingOrPhotograph"

They’re clearly not “human observations” in the way that makes sense to most (i.e. “I saw a Salmon Gum 5 km S of Coolgardie yesterday”) – the photocopies are of actual plant specimens that can be physically studied (you can request them from the regional herbarium in which they’re held); they’re physical placeholders – but as photocopies they are best represented by the TDWG definition of humanObservation as a record type (i.e. “an output of a human observation process”). Yuck…

Cheers, Ryonen

**Comment by nielsklazenga** [comment]
It doesn't matter what basisOfRecord you put on it: photocopies and cibachromes should not be delivered to AVH.

**Comment by ryonen** [comment]
Okay – a bit more info for the newbie please, and maybe some suggestions as to how this gets resolved?

To reiterate, as food for thought - the photocopies at PERTH are of specimens that belong to the Western Australian Herbarium but are housed elsewhere. In some cases there may be less than five collections of a taxon in existence, with two of those collections held in (e.g.) Albany (i.e. at ALB.) but represented at PERTH as barcoded photocopies so that the specimen information can be made available electronically (FloraBase, AVH). Clearly that data is important and needs to be made available. If a researcher from MEL or elsewhere wants to look at all of the specimens of that taxon they can; PERTH will organise for the two specimens at ALB. to be sent to PERTH and loaned to MEL, along with whatever PERTH physically holds. This situation is less than ideal, but Western Australia is 1/3 of the Australian landmass and only has one IK-registered herbarium. The data for the specimens not physically held at PERTH still needs to be delivered.

**Comment by nielsklazenga** [comment]
Ah, those are of course perfectly fine in AVH. In this situation I would probably deliver the acronym of the herbarium where the specimen is housed ('ALB' in your example) as the collectionCode (SourceID in ABCD) and the basisOfRecord just as 'PreservedSpecimen'. This is basically saying the specimen is in a different collection that also belongs to PERTH.

You could also say that AVH users don't need to know that the specimen is not housed in PERTH, but somewhere else, and just deliver basisOfRecord as 'PreservedSpecimen'.

basisOfRecord for anything that is derived from a preserved specimen (photocopy, cibachrome, microscope slide etc.) is always 'PreservedSpecimen'. This is even true if the preserved specimen doesn't exist anymore or can't be located. You can indicate this in the [dwc:disposition](http://hiscom.github.io/hispid/terms#disposition) field (you could also use this field to indicate that the specimen is housed elsewhere, if we extend the vocabulary). You can use the [dcterms:type](http://hiscom.github.io/hispid/terms#type) field to indicate that the thing the record is from is an 'Image', as opposed to a 'PhysicalObject'.

Lots of options.

**Comment by ryonen** [comment]
Thanks Niels! I’ve had a chat with Karina and there are definitely some solutions in there. We’ll raise this with Ben and with Julia Percy-Bowers (our collections manager) and see what course of action is best. It might take a little while to fix, but it will be.

Now… on to CANB about their >3,500 HumanObservations…

**Comment by ben3000** [comment]
This is possibly due to a misconception of mine regarding basisOfRecord. Niels, could you please point me to documentation that supports your statement that "basisOfRecord for anything that is derived from a preserved specimen [..] is always 'PreservedSpecimen'"?

**Comment by nielsklazenga** [comment]
It is just logic. With the ABCD vocabulary (of RecordBasis) it might be debatable, because that has PhotographOrDrawing and MultimediaObject, but in the Darwin Core vocabulary it is clear. 

It is actually irrelevant. In Darwin Core RDF basisOfRecord is not used, as in RDF you just have evidence for an Occurrence (through the hasEvidence property in DwC-SW; I think Filtered Push has it as well). What is pertinent here is that, if you deliver PhotographOrDrawing, ALA will interpret it as HumanObservation (which I believe is correct mapping-wise). If you don't want your records to show as HumanObservation, you should deliver basisOfRecord as PreservedSpecimen or not deliver basisOfRecord at all.

We should probably have moved this discussion into a new issue a while back.

**Comment by ben3000** [comment]
I think I found an answer at http://rs.tdwg.org/dwc/terms/:

> Terms under the Record-level Terms section apply to the whole record regardless of the record type. For example, a record for a camera trap bird sighting would fall into the Dublin Core dcterms:type category StillImage and at the same time into the Darwin Core basisOfRecord category MachineObservation - a particular type of Occurrence [..]

`basisOfRecord` is a Record-level term.

**Comment by nielsklazenga** [comment]
Ah, good, Wouldn't have thought to look there. So I wasn't just dreaming it up.

**Comment by ben3000** [comment]
No, and now that I keep reading further, you said the same yourself: https://github.com/hiscom/hispid/issues/27#issue-56147433.
